python/pyspark/sql/tests.py

```diff
@@ -30,6 +30,19 @@
 import functools
 import time
 import datetime
+import traceback
+
+if sys.version_info[:2] <= (2, 6):
```

Comment:
Actually, this should be removed. These are duplicated imports below. I don't know why it was added in that PR.